pom.xml

```diff
@@ -34,6 +34,17 @@
             <groupId>com.github.onsdigital</groupId>
             <artifactId>restolino</artifactId>
             <version>v0.5.0</version>
+             <exclusions>
+                <exclusion>
+                    <groupId>commons-io</groupId>
+                    <artifactId>commons-io</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Do you need this exclusion? restolino brings in commons-io:2.8 which is ommitted anyway as 2.9 is greater. Also we could upgrade restolino to 2.9 instead and avoid the direct dependency upgrade https://github.com/ONSdigital/restolino/blob/master/pom.xml#L73